docs/usage.md

```diff
@@ -0,0 +1,500 @@
+# Usage
+
+:warning: This file shows example commands for reading, creating, updating and deleting redmine components. As this are only examples, please **make sure, that you do not execute them at your production Redmine instance and may destroy real life content!**
```

Comment:
```suggestion
:warning: This file shows example commands for reading, creating, updating and deleting Redmine data. As these are only examples, please **make sure, that you do not execute them against your production instance as it may break real content!**
```